Userland/Libraries/LibGUI/AbstractThemePreview.h

```diff
@@ -31,6 +31,13 @@ class AbstractThemePreview : public GUI::Frame {
     Function<void(String const&)> on_theme_load_from_file;
     Function<void()> on_palette_change;
 
+    struct Window {
+        Gfx::IntRect& rect;
+        Window(Gfx::IntRect& rect)
+            : rect { rect } {};
```

Comment:
Do we need an explicit ctor for this?

Reply:
Seems not, don't know why I assumed we did